async/format.browser.js

```diff
@@ -1,5 +1,8 @@
 module.exports = function (random, alphabet, size) {
+  // TODO
   var mask = (2 << 31 - Math.clz32((alphabet.length - 1) | 1)) - 1
```

Comment:
Should I add anything for masks?
Below as well

Reply:
1. We need to refuse numbers bigger than the alphabet. You can add this note to `id += alphabet[bytes[i] & mask] || ''` as well, `|| ''` is to refuse byte if it is bigger than `alphabet.length`
2. To refuse fewer numbers we apply bitmask. For instance, we got to have 8 symbols abt alphabet and random generator gave us `9` (`101`). Without mask hack, we need to refuse this byte. But we apply `11` mask on it and got `001` (`1`).
3. It doesn’t solve all problems, because bitmask can reduce big bytes to closed `x^^2` number (if we have 7 symbols in the alphabet, bitmask will pass `8` anyway).

Reply:
Done ✅